markov/experimental.R

```diff
@@ -0,0 +1,431 @@
+# Collecting experimental results
+
+library(singlearity)
+suppressPackageStartupMessages(library(tidyverse))
+suppressPackageStartupMessages(library(baseballr))
+suppressPackageStartupMessages(library(lubridate))
+
+# Assumes user is in the overarching "Singlearity" directory 
+
+source(file = "markov/markov.R")
+source(file = "R/get_singlearity_client.R")
```

Comment:
You shouldn't need to include get_singlearity_client.R anymore.  You may need to update to singlearity 0.1.8+ to be able to leave this out.

This same comment applies to any other places where you've included get_singlearity_client.R